tests/components/jewish_calendar/conftest.py

```diff
@@ -26,3 +39,78 @@ def mock_setup_entry() -> Generator[AsyncMock]:
         "homeassistant.components.jewish_calendar.async_setup_entry", return_value=True
     ) as mock_setup_entry:
         yield mock_setup_entry
+
+
+@pytest.fixture
+def jcal_params(request: pytest.FixtureRequest) -> dict | None:
+    """Return test parameters."""
+    if not hasattr(request, "param"):
+        return None
+
+    if len(request.param) == 3:
+        location_name, test_time, results = request.param
```

Comment:
Having multiple items in the request params makes tests hard to follow. Consider making actual fixtures for each of them.